### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by torsteingrindvik** [TimJentzsch/bevy_mod_krita#13] [General]
Hi, I'm interested in Krita+Bevy use too.

I was wondering:

1. Are you open to PRs to keep this plugin up to date?
2. Or would you rather do it yourself?

If neither then I will likely fork/remake but I'm happy to make PRs to yours if you'd like.

**Comment by TimJentzsch** [comment]
Thanks for your interest in this plugin.
Yes, I'm open for PRs. I might have some time myself to update to 0.12 this week, but I also wouldn't mind someone else to take a shot at this.

I think 0.12 is great for this plugin: Instead of having to pay the cost of bundling big `.kra` files in the game, we could levarage the new asset pre processing to instead include leaner image files, while still having the simplicity of Krita files in the dev workflow for fast iterations.
Perhaps this can make this plugin useful beyond just game jams and prototypes.

If you decide to take on a task, please let me know beforehand, in order to avoid duplicate work.
Thank you!

**Reply by torsteingrindvik** [reply]
Ah- I haven't looked that much into the processing parts of asset v2 yet.
I'll observe the repo, and I won't start anything without pinging here.

Thanks in advance.
